openmc/geometry.py

```diff
@@ -58,6 +58,16 @@ def __init__(
                     univ.add_cell(cell)
                 self._root_universe = univ
 
+    def __contains__(self, cell):
+
+        try:
+            check_type('cell', cell, openmc.Cell)
+        except:
+            return False
```

Comment:
I don't think it's appropriate to silently suppress a `TypeError`. This shouldn't be in a try-except block IMO.